integration/v7/isolated/continue_deployment_test.go

```diff
@@ -23,22 +23,21 @@ var _ = Describe("Continue Deployment", func() {
 
 		It("displays the help information", func() {
 			session := helpers.CF("continue-deployment", "--help")
-			Eventually(session).Should(Say(`NAME:`))
-			Eventually(session).Should(Say(`continue-deployment - Continue the most recent deployment for an app.\n`))
-			Eventually(session).Should(Say(`\n`))
```

Comment:
**NOTE**

I've consolidated these multiple assertions into a single multiline assertion. The previous version suggests that it's asserting on the complete contents of the `--help` output but it's actually just checking that the entire string contains, e.g., a `\n` *anywhere in the output*.

Reply:
Ok and now I've reverted these changes, as it's been pointed out to me that `session` is `Buffer` and not a `string`, so it actually *does* (more or less) assert on the full contents of the help message.